HeroVault/HeroVault/Features/UserProfile/UserProfileViewController.swift

```diff
@@ -9,7 +9,29 @@ import UIKit
 
 class UserProfileViewController: UIViewController {
     
+    // MARK: IBOutlets
+    
+    @IBOutlet weak private var profileLabel: UILabel!
+    @IBOutlet weak private var userImage: UIImageView!
+    @IBOutlet private weak var userNameLabel: UILabel!
+    @IBOutlet private weak var favouriteCharactersLabel: UILabel!
```

Comment:
private in wrong place. should be weak private var😄 Did your code run? It should have broken when you tried to go to this screen. Make sure to always check your code builds and runs before making a pr😄

Reply:
The code did run. I've updated this code, thanks.